test/test_list_opening.vader

```diff
@@ -33,16 +35,29 @@ Before:
     return 0
   endfunction
 
+  " If the window is vertical, window size should match column size/width
+  function GetQuickfixIsVertical(cols) abort
+    for l:win in range(1, winnr('$'))
+        if getwinvar(l:win, '&buftype') ==# 'quickfix'
```

Comment:
`is#` is preferred in this codebase when checking strings because `==` behaves weirdly with strings and numbers.